modules/chemical_reactions/doc/content/syntax/ChemicalComposition/index.md

```diff
@@ -0,0 +1,33 @@
+# ChemicalComposition Action System
+
```

Comment:
since you re modifying things, you should link to the action object in some way here

it s typical to have all the lists of objects in these syntax pages?

Reply:
Will add this in a follow on PR